### PR TITLE
[Ready for Review] Updating README (ITA)

### PR DESCRIPTION
**Pull request by perluna**
### Summary

Working on #2051

### List of changes proposed in this PR (pull-request)

* The table and the images (at the top) are from the ENG readme with minimal changes to Italian, wherever possible.
* Big changes/additions to the top section before the index (with images), "the team" section and the "contribution" one
* All the README is now updated 
* Updated number of contributors in the badge as of 2021-08-15 3.30pm

### What should a reviewer concentrate their feedback on?

- [ ] Unsure about the path to images, may need help/suggestions on this, please. (I used the full path to to the images, unsure if it's ok)
- [ ] Most of the doc is translated pretty literally (as it's also easier to update) but line 84 of the original README is translated focusing on the meaning. There may be a better translation to be closer to ENG. The final result is in places a bit clunky and may appear a tad weird to Italian-speaking readers but the meaning is there.
- [ ] I'd argue that "repository" is masculine, rather than feminine, in Italian - after the meaning of "archive", "deposit" or even "database" which are all masculine. Unsure if there was a reason for the feminine.

### Acknowledging contributors

- [ ] All contributors to this pull request are already named in the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/master/README.md#contributors) in the README file.
- [x] The following people should be added to the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/master/README.md#contributors) in the README file: @perluna

### CODE REVIEW

**Comment by malvikasharan** [comment]
Thanks for the PR @perluna, let me know when you are ready for suggestions on reviewers. Meanwhile, I will add you to the contributor's list.

**Comment by malvikasharan** [comment]
@all-contributors please add @perluna for translation

**Comment by perluna** [comment]
@malvikasharan this is now ready for review, thanks!

**Comment by fedenanni** [comment]
@malvikasharan ehi! let me know if I can help with the review :D

**Comment by BatoolMM** [comment]
I don't wanna speak on behalf of Malvika but I'm sure @malvikasharan will be very happy if you can review it @fedenanni.

File: README-translated/README-Italian.md [hunk 1]
```diff
@@ -1,20 +1,46 @@
...
+
+_The Turing Way_ è una guida per condurre data science in una maniera riproducibile, etica e collaborativa. Coinvolgiamo e sosteniamo una comunità eterogenea di contributori in modo da rendere data science accessibile, comprensibile ed efficace per tutti. Il nostro obiettivo è quello di fornire tutte le informazioni di cui i ricercatori e i data scientists (che siano in ambito accademico, industriale o nel settore pubblico) hanno bisogno all'inizio dei loro progetti in modo che questi siano poi facili da riprodurre.
+
+[![The Turing Way project è un libro, una comunità, un progetto open-source e una cultura collaborativa. Questo è illustrato in quattro figure. La prima mostra il libro "the Turing Way", la seconda spiega come la comunità può crescere, la terza figura rappresenta due persone che collaborano per una pull request, l'ultima illustra l'equilibrio dove la reproducibilità è considerata più importante del numero di articoli pubblicati.](https://github.com/alan-turing-institute/the-turing-way/blob/master/book/website/figures/README_imgs/README_turingway.png)](https://docs.google.com/presentation/d/13Nm8LcRW87ffxEugGEs5j6HKQEvhiuH8b7MfIdX7MpI/edit#slide=id.p1).
```
Comment (fedenanni) [inline README-translated/README-Italian.md:26]: Regarding the path to images - I would follow what people have done in other README and use local paths, like `book/website/figures/README_imgs/README_turingway.png`

File: README-translated/README-Italian.md [hunk 2]
```diff
@@ -24,38 +50,49 @@ Indice:
...
+
+Consulta [l'elenco dei collaboratori](#contributors) per i profili GitHub di tutti i contributori.
 
 ### Come contribuire
 
-:construction: Questa repository è sempre in costruzione e **tutti** sono incoraggiati ad aiutarci in questo progetto che può essere utile a molti. :construction:
+:construction: Questo repository è sempre in costruzione e **tutti** sono incoraggiati ad aiutarci in questo progetto che può essere utile a molti. :construction:
+
+Tutti sono tenuti a seguire il nostro [codice di condotta](CODE_OF_CONDUCT.md) e a leggere le nostre [linee guida](CONTRIBUTING.md) per avere più informazioni su come iniziare.
+Vogliamo rendere la contribuzione a questo progetto il più facile possibile.
```
Comment (fedenanni) [inline README-translated/README-Italian.md:80]: I think the translation works great and keeps the meaning!

File: README-translated/README-Italian.md [hunk 3]
```diff
@@ -24,38 +50,49 @@ Indice:
...
+Ai membri della comunità vengono date opportunità per imparare nuove abilità, condividere le proprie idee e collaborare con altri.
+Vengono anche fornite possibilità di avere, e si incoraggia poi ad essere, un mentore per contribuzioni a _The Turing Way_ e ad altri progetti open source.
+
+Abbiamo creato un [promotion pack](https://github.com/alan-turing-institute/the-turing-way/tree/master/communications/promotion-pack) per aiutarti a presentare e condividere _The Turing Way_.
 
 ### Citare _The Turing Way_
 
 Puoi citare _The Turing Way_ attraverso l'archivio del progetto Zenodo usando il DOI: [10.5281/zenodo.3233853](https://doi.org/10.5281/zenodo.3233853).
-Il DOI (digital object identifier) ci permette di archiviare le repository ed è molto utile per garantire che il lavoro venga tracciato nelle pubblicazioni accademiche.
+Il DOI (digital object identifier) ci permette di archiviare i repository ed è molto utile per garantire che il lavoro venga tracciato nelle pubblicazioni accademiche.
```
Comment (fedenanni) [inline README-translated/README-Italian.md:95]: Regarding `repository`, from [l'Accademia della Crusca](https://accademiadellacrusca.it/it/consulenza/quale-possibile-traducente-per-emrepositoryem/2859) 🧐  :

> Le attestazioni mostrano una netta prevalenza per il genere maschile invariabile (il repository/i repository, che ottengono su Google rispettivamente 93.600 e 37.800 risultati), anche se, come spesso succede per i prestiti inglesi, si registra una certa oscillazione nell’attribuzione del genere (le occorrenze per il femminile sono 8.910 per la repository e 3.750 per le repository) e del numero (le attestazioni per la forma flessa sono 3.590 per i repositories e 831 per le repositories). Generalmente il genere di un prestito viene assegnato in base al genere del nome che viene individuato o percepito come traducente: in questo caso gli equivalenti italiani più in uso sono archivio, magazzino, deposito, tutte forme maschili. L’uso del femminile non si può dunque spiegare con tale criterio: è invece il suffisso -ory (di factory, memory, ecc.) a indirizzare verso il genere femminile.

 Both would work for me, but the masculine is way more common.

**Review by fedenanni — COMMENTED** [review]
Hi @perluna ! I just gave it a first read right before lunch and left a couple of comments regarding the main points you highlighted in your comment. I'll give it a second pass later today and leave some suggestions if the translated flow can be improved, but at a first read it works nicely!

File: README-translated/README-Italian.md [hunk 4]
```diff
@@ -1,20 +1,46 @@
...
+| **Cita** |  [![Cite this using DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.3233853.svg)](https://doi.org/10.5281/zenodo.3233853) |
+
+
+_Benvenuti nel GitHub repository per The Turing Way. Qui è dove tutte le parti del progetto vengono sviluppate, riviste e mantenute._
```
Comment (fedenanni) [inline README-translated/README-Italian.md:22]: Not sure but "mantenute" here sounds a bit too much as a direct translation. Maybe "sostenute" but I am not sure

File: README-translated/README-Italian.md [hunk 5]
```diff
@@ -1,20 +1,46 @@
...
+
+_The Turing Way_ è una guida per condurre data science in una maniera riproducibile, etica e collaborativa. Coinvolgiamo e sosteniamo una comunità eterogenea di contributori in modo da rendere data science accessibile, comprensibile ed efficace per tutti. Il nostro obiettivo è quello di fornire tutte le informazioni di cui i ricercatori e i data scientists (che siano in ambito accademico, industriale o nel settore pubblico) hanno bisogno all'inizio dei loro progetti in modo che questi siano poi facili da riprodurre.
```
Comment (fedenanni) [inline README-translated/README-Italian.md:24]: Maybe "in maniera riproducibile"

File: README-translated/README-Italian.md [hunk 6]
```diff
@@ -1,20 +1,46 @@
...
+
+[![Questo è uno screenshot del libro online. Mostra una delle illustrazioni all'inizio del libro. In questa figura, c'è una strada dove ci sono negozi per le varie data science skills. I clienti, con il loro carrello, possono entrare e uscire a loro piacimento e scegliere ciò che gli serve.](https://github.com/alan-turing-institute/the-turing-way/blob/master/book/website/figures/README_imgs/README_book.png)](https://the-turing-way.netlify.app/welcome.html)
+
+*Screenshot del libro The Turing Way online ([usa questa immagine in una presentazione](https://drive.google.com/file/d/1wJR664YECSc8b_RSHeyVjDlHs-Ls9lkc/view?usp=sharing))*
+
+Il libro _The Turing Way_ è iniziato come una guida lievemente opinionata nel 2019 e si è gradualmente trasformato in una serie di guide su Ricerca Riproducibile, Design di Progetti, Comunicazione, Collaborazione e Ricerca Etica.
```
Comment (fedenanni) [inline README-translated/README-Italian.md:37]: "lievemente opinionata" sounds a bit strange. maybe "con un taglio deciso"? Not sure if it keeps the original meaning though

File: README-translated/README-Italian.md [hunk 7]
```diff
@@ -24,38 +50,49 @@ Indice:
...
 ### Come contribuire
 
-:construction: Questa repository è sempre in costruzione e **tutti** sono incoraggiati ad aiutarci in questo progetto che può essere utile a molti. :construction:
+:construction: Questo repository è sempre in costruzione e **tutti** sono incoraggiati ad aiutarci in questo progetto che può essere utile a molti. :construction:
+
+Tutti sono tenuti a seguire il nostro [codice di condotta](CODE_OF_CONDUCT.md) e a leggere le nostre [linee guida](CONTRIBUTING.md) per avere più informazioni su come iniziare.
+Vogliamo rendere la contribuzione a questo progetto il più facile possibile.
+Per questo motivo, ci sono diversi modi in cui è possibile contribuire in base ai propri interessi, disponibilità e capacità.
+
+![Questa immagine mostra sei dei diversi modi in cui chiunque può contribuire. Questi sono: sviluppare e condividere, mantenere e migliorare, condividere risorse, revisione e aggiornamento, rendere il progetto globale attraverso la traduzione, e condividere le migliori pratiche.](https://github.com/alan-turing-institute/the-turing-way/blob/master/book/website/figures/README_imgs/README_contributions.png)
```
Comment (fedenanni) [inline README-translated/README-Italian.md:83]: given that all other are verbs at the infinite tense, you could have "revisionare e aggiornare"?

File: README-translated/README-Italian.md [hunk 8]
```diff
@@ -82,18 +119,30 @@ L'ultima versione di Zenodo può essere menzionata come:
...
+Abbiamo utilizzato alcune di queste immagini nelle risposte del [Welcome Bot](https://github.com/apps/welcome) ai contributi dei nuovi membri nel nostro repository su GitHub.
 
 ### Contattaci
 
-Abbiamo una chat room su gitter e ci farebbe piacere se passassi a dire ciao <https://gitter.im/alan-turing-institute/the-turing-way>.
+#### Email
+
+Puoi contattare il team *The Turing Way* scrivendo a [theturingway@gmail.com](mailto:theturingway@gmail.com).
+
+In alternativa, puoi contattare **Malvika Sharan** tramite [msharan@turing.ac.uk](mailto:msharan@turing.ac.uk) oppure **Kirstie Whitaker** scrivendo a [kwhitaker@turing.ac.uk](mailto:kwhitaker@turing.ac.uk).
```
Comment (fedenanni) [inline README-translated/README-Italian.md:130]: Maybe you could say to get in touch with them in English

**Review by fedenanni — COMMENTED** [review]
@perluna thank you again! I gave it now a full read and it works nicely! I left a couple of suggestions, but apart from the thing about the paths of the images, which you could make it local, all the rest are very small details - ciao!

File: README-translated/README-Italian.md (hunk 4 above)
Comment (perluna) [reply README-translated/README-Italian.md:22]: what do you think of "tenute aggiornate"?

File: README-translated/README-Italian.md (hunk 6 above)
Comment (perluna) [reply README-translated/README-Italian.md:37]: like this option! Agree, my translation was a bit weird

File: README-translated/README-Italian.md (hunk 8 above)
Comment (perluna) [reply README-translated/README-Italian.md:130]: good point. I added it at a heading level as all emails/chats will be in English. Reckon it works this way?

**Comment by perluna** [comment]
Thanks @fedenanni for the review! I do agree with basically all your changes so just did a commit to fix those as well as the images paths

File: README-translated/README-Italian.md [hunk 9]
```diff
@@ -24,38 +50,49 @@ Indice:
...
 
-Tutti sono tenuti a seguire il nostro [codice di condotta](CODE_OF_CONDUCT.md) e a leggere le nostre [linee guida](CONTRIBUTING.md) prima di iniziare.
+*Tra i modi di contribuire ci sono anche lo sviluppo e la condivisione di nuovi capitoli, la manutenzione e il miglioramento dei capitoli esistenti, la condivisione delle risorse de _The Turing Way_, la revisione e l'aggiornamento del materiale, la traduzione dei capitoli per rendere questo progetto accessibile globalmente, la condivisione delle miglior pratiche nell'amito della ricerca.*
 
-Se non te la senti di aiutare su GitHub, potresti anche contribuire con un caso di studio o con dei suggerimenti tramite il nostro [form su Google docs](https://goo.gl/forms/akFqZEIy2kxAjfZW2).
+Ai membri della comunità vengono date opportunità per imparare nuove abilità, condividere le proprie idee e collaborare con altri.
+Vengono anche fornite possibilità di avere, e si incoraggia poi ad essere, un mentore per contribuzioni a _The Turing Way_ e ad altri progetti open source.
```
[the remark this answers is not in the document]
Comment (perluna) [reply README-translated/README-Italian.md:88]: tough spot here but what about changing it to "Questa comunità rende possibile per i propri membri, e li incoraggia, a fare da guida per contribuzioni a The Turing way e ad altri progetti open source."?

File: README-translated/README-Italian.md (hunk 4 above)
Comment (fedenanni) [reply README-translated/README-Italian.md:22]: works great!

File: README-translated/README-Italian.md (hunk 9 above)
[the remark this answers is not in the document]
Comment (fedenanni) [reply README-translated/README-Italian.md:88]: yes!

File: README-translated/README-Italian.md (hunk 8 above)
Comment (fedenanni) [reply README-translated/README-Italian.md:130]: great!

**Review by fedenanni — APPROVED** [review]
@perluna Looks great to me! @BatoolMM @malvikasharan You can go ahead and merge it ;-)

**Review by BatoolMM — APPROVED** [review]
Thank you so much @perluna for your work and @fedenanni for reviewing the PR and making valuable comments 🌸. @malvikasharan feel free to merge if you are happy with the changes made in the PR!

**Comment by BatoolMM** [comment]
@all-contributors please add @fedenanni for review

**Comment by malvikasharan** [comment]
Can someone help with the conflict of interest? @perluna ?

**Comment by gcroci2** [comment]
I've solved the conflict :)

**Comment by malvikasharan** [comment]
@all-contributors please add @gcroci2 for translation and review

**Review by malvikasharan — APPROVED** [review]

**Pull request merged**